test/Docker.DotNet.Tests/ISystemOperations.Tests.cs

```diff
@@ -21,31 +22,54 @@ public ISystemOperationsTests()
         public void DockerService_IsRunning()
         {
             var services = ServiceController.GetServices();
-            using (var dockerService = services.SingleOrDefault(service => service.ServiceName == "docker"))
+            using (var dockerService = services.FirstOrDefault(service => service.ServiceName == "docker" || service.ServiceName == "com.docker.service"))

```

Comment:
`ServiceController` is not cross platform. We should find another way to deal with this. Perhaps have a common fixture that uses `Process` to check if Docker is running or not. This will definitively throw `PlatformNotSupportedException` on non-Windows machines and is also is deprecated on .Net 5 (which I intend to migrate soon).

Reply:
@galvesribeiro this test isn't executed in Windows platform because of the [SupportedOSPlatformsFact(Platform.Windows)] attribute. 
I agree, the propper thing would be for the test case to work on all supported platforms, but right now I have no idea on how to do it.
Do you have any suggestion? I'd appreciate that

Reply:
Sorry, I've only looked to the snipped that changed.

You can replace the `ServiceController.GetServices()` with something like `Process.GetProcesses().FirstOrDefault(service => ....)`.

Reply:
Done